TangemStaking/Services/StakeKitMapper/StakeKitMapper.swift

```diff
@@ -246,6 +246,9 @@ struct StakeKitMapper {
 
         let validators = response.validators.map(mapToValidatorInfo)
         let preferredValidators = validators.filter { $0.preferred }.sorted { lhs, rhs in
+            if lhs.name.caseInsensitiveCompare(StakingConstants.partnerName) == .orderedSame {
+                return true
+            }
             if lhs.apr == rhs.apr {
                 return lhs.partner
             }
```

Comment:
а у нас уже заполняется `StakeKitMapper.mapToValidatorInfo` поле `partner` По факту можно просто его в перед поставить. По этому флагу, что то типо

```suggestion
            if lhs.partner {
                return true
            }
```

Reply:
поправил, спасибо!